cypress/e2e/ts-test/ts-test.cy.ts

```diff
@@ -1,5 +1,6 @@
 describe("login page", () => {
     it("should visit login page", () => {
         cy.visit("www.google.com");
+        cy.get("#APjFqb").type("something");
```

Comment:
I dont this is necessary it will make complications in future. since its not referring any code standards.

- [x] remote unnecessary code

Reply:
thats actually very good point. Duplicates has been removed.